cmd/ak/cmd/runtimes/test.go

```diff
@@ -0,0 +1,87 @@
+package runtimes
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/hexops/gotextdiff"
+	"github.com/hexops/gotextdiff/myers"
+	"github.com/hexops/gotextdiff/span"
+	"github.com/spf13/cobra"
+	"golang.org/x/tools/txtar"
+
+	"go.autokitteh.dev/autokitteh/cmd/ak/common"
+	"go.autokitteh.dev/autokitteh/internal/kittehs"
+	"go.autokitteh.dev/autokitteh/sdk/sdkbuildfile"
+)
+
+var testCmd = common.StandardCommand(&cobra.Command{
+	Use:     "test <txtar-path> [-timeout t] [--quiet]",
+	Short:   `Test a program`,
+	Aliases: []string{"t"},
+	Args:    cobra.ExactArgs(1),
+
+	RunE: func(cmd *cobra.Command, args []string) error {
+		f, err := os.OpenFile(args[0], os.O_RDONLY, 0)
+		if err != nil {
+			return fmt.Errorf("open: %w", err)
+		}
+
+		defer f.Close()
+
+		var b *sdkbuildfile.BuildFile
+
+		var expectedPrints string
+
+		bs, err := io.ReadAll(f)
+		if err != nil {
+			return fmt.Errorf("read: %w", err)
```

Comment:
No need to prefix IO errors: they are very self-descriptive

Reply:
done